types/validation.go

```diff
@@ -427,3 +451,96 @@ func verifyBasicValsAndCommit(vals *ValidatorSet, commit *Commit, height int64,
 
 	return nil
 }
+
+func verifyAggregatedCommit(
+	chainID string,
+	vals *ValidatorSet,
+	commit *Commit,
+	votingPowerNeeded int64,
+	ignoreSig func(CommitSig) bool,
+	countSig func(CommitSig) bool,
```

Comment:
I don't think `ignoreSig` and `countSig` make any sense when dealing with an aggregate signature

Reply:
OK, I thought a bit about this. I think we just need to pass a boolean denoting whether we want to verify the `nil` aggregated signature. That boolean should be:
* true in `VerifyCommit`
* false in all light client verification calls

Reply:
agree [`141d1a2` (#11)](https://github.com/berachain/cometbft/pull/11/commits/141d1a2859e9a21ae01cf4f54cbc4aafc9b62d7a)